### PR TITLE
LPD-27633 Remove site breadcrumb when moving categories in Message Boards

### PR DESCRIPTION
**Pull request by SamZiemer**
# What is this trying to solve?
https://liferay.atlassian.net/browse/LPD-27633

I am removing the site from the breadcrumbs when moving a category in message boards.

# How am I fixing it?
Passing in the correct booleans when getting the breadcrumbs

# How can you verify that it works?
Run npm run playwright test -- -g 'LPD-27633' to run the test

### CODE REVIEW

**Comment by SamZiemer** [comment]
ci:test:relevant

**Comment by SamZiemer** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
@SamZiemer The mb playwright tests are failing. Can you take a look?

**Comment by SamZiemer** [comment]
Thank you @adolfopa, I'm looking into what is causing this failure.

File: modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts [hunk 1]
```diff
@@ -48,3 +49,35 @@ test('LPD-25630 Show the status to guest user', async ({
 
 	await expect(page.getByText('Pending')).toBeVisible();
 });
+
+test('LPD-27633 Do not show site in breadcrumb', async ({
+	messageBoardsPage,
+	messageBoardsWidgetPage,
+	page,
+	site,
+	workflowPage,
```
Comment (marcogalluzzi) [inline modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts:58]: Please remove unused parameters `messageBoardsPage` and `workflowPage`.

File: modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts [hunk 2]
```diff
@@ -48,3 +49,35 @@ test('LPD-25630 Show the status to guest user', async ({
...
+}) => {
+	await page.goto(site.friendlyUrlPath);
```
Comment (marcogalluzzi) [inline modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts:60]: This `page.goto` navigates to an unknown page (404). It isn't needed.

File: modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts [hunk 3]
```diff
@@ -48,3 +49,35 @@ test('LPD-25630 Show the status to guest user', async ({
...
+
+	const categoryName = getRandomString();
+
+	await messageBoardsWidgetPage.addCategory(site, layout, categoryName);
+
+	await page
+		.locator(
+			'[id="_com_liferay_message_boards_web_portlet_MBPortlet_mbCategoriesSearchContainer_1_menu"]'
+		)
+		.click();
```
Comment (marcogalluzzi) [inline modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts:72]: This instruction was randomly failing. I added a `waitFor` and it seems to work better.

```javascript
	const searchMenu = page
		.locator(
			'[id="_com_liferay_message_boards_web_portlet_MBPortlet_mbCategoriesSearchContainer_1_menu"]'
		);

	await searchMenu.waitFor();
	await searchMenu.click();
```

**Review by marcogalluzzi — CHANGES_REQUESTED** [review]
Hi @SamZiemer I couln't reproduce the CI error locally. Hopefully removing the unuseful initial `goto` instruction make it working fine.

File: modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts (hunk 3 above)
Comment (SamZiemer) [reply modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts]: Thank you Marco, I've made those changes and I'll run the tests again to see if it helped.

**Comment by SamZiemer** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:playwright

**Comment by SamZiemer** [comment]
ci:test:playwright

**Comment by SamZiemer** [comment]
It looks like these new failures are not related to my changes.

**Comment by SamZiemer** [comment]
ci:test:relevant

**Review by ambrinchaudhary — APPROVED** [review]
The wait.for() may not be enought. It failed first time in Ci but after the retry it passed, see https://ff50992fd4a999f47081b33323fe77b29e54ba69d87f787fbec68a0-apidata.googleusercontent.com/download/storage/v1/b/testray-results/o/2024-06%2Ftest-1-33%2Ftest-portal-acceptance-pullrequest(master)%2F6292%2Fplaywright-js-tomcat90-mysql57-jdk8%2F20%2F0%2Fplaywright-report%2Findex.html?jk=AS1gM8Nt4zVO6hohV6tthzlLu0K6hzsbmntqjV79ybFyoNGlwSnGCIKmnQno4B-ZdMEDkB6W_5BKq_r7TSpXd1HpPA-3vyvj64HQc0egbGEaCOSV6Pj70YHirolZQZdXFcclQ0Iug6D-4UdBMLkhCS3kEcHa6FDdDAhha9gouHRd4AVltBYN0jz5qCKm4-NbdgZx5AvRkqyZYHBkAgC_1i-Op-xtvi27HDV3m-V_ZX_VP8tk9_k0JCWTnBc-wrJZlgEpCepe7uCm1OzoBvDQyIfJH5cPsiUJGS2qyqUTmLowcFM8TVhrUrPg49wzt4nR_d3lRqiq4fx9vs9zyBFZR4ka4sCJb3AVyOtuhqmTSVINf2kq_ilobSvAV6AtEJmv02e4UmTTQlU_IpD71rHKKpKX7T0bCfvFAopKflwLkacdcYRRVW_4ZK73UoL6D7mH60OYBKF9YePA9reVWp9ai96BTTgCtg6DzDem_ZFbHYLcxLrHtL-sM6oN9ogz9g2MT97joAH7I2M5zAELAhBwwuD1VIxl5xTTDRFNbLXpEi3a38NwWaDJ_u4huH35LfocLSv0wDdt9jrt4ugCf791YdnOfHD5dKfifyOuLK4_azYYrYC-L8Jlj-fGKUfGBA-nAtiuAuN9YU1EhMLwvaChw0m6PAoU7l-dIc55bGMvAUCBmBRFqyo4OrAEpAvuj3BTT8mSkaLuiQOzBLWjXx0YTlkdl-2Q_WRyXC80zIoDuwhpOLb71YdfnC_m1DS2Q3Z8ZU5TSVCve8wouBq0yIp7Ixd9VVWHS0UiywAxVLy1FuK3DU5kcT-cUmDYqAZawScMXyNG9whzvjxA6QqmXhNY9r3xLEb1IsWgFfWZXo1hOhsN-Q_XdZ0NxTRJl1GZbCjXO2CkrdKWqZcR7MNMTDucu9SocndankAOLM99zXSoPPE-98LUTEfc6t56BcFqixodWYAgLVP1j8TAIRRAC2mwBUJ-RsvzQ5Mk0ymLqAuE5RQGWgvRwIhvA_hKc_08O6Eg3OPCeb_gYULTJ02-5Lj---x7J_vngoPC8714cngtod9CqhmbDHuWi0PlNt-XR7iMgyZvYUgRt_raLj-d4OEHzinoNu_x8SszLTRYELjrArfec5s509IV_Hyr4-nM11L76viy4wy6yMIDt3lvruXr0Lu3CUXieYlFt2d-kcmQ0AM5Nr8Q0Qo93IMuqqoA_UL1j3_ml7UtPhqCVQKX0rAEkI5kWzg62Z-M71Q2fn48yRHdrapAdAHymkFY7trPPagOQXRRjAoXzAdLS_XEgMftmqw1KAVXQGZ5qmjalJMLXMg16LTaKo-8oc0S_642_ltOg6-i7vu6z3BnZrGU4hW3SO8&isca=1#?testId=c2a76291138eb039aa6d-6bf58cfe4f8807ffd177 

Let's move this forward and keep an eye in future runs to determine if this is flaky or not.

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**